packaging/installer/functions.sh

```diff
@@ -727,3 +727,64 @@ safe_sha256sum() {
 		fatal "I could not find a suitable checksum binary to use"
 	fi
 }
+
+install_or_remove_netdata_updater() {
+	if [ -f "${NETDATA_PREFIX}"/usr/libexec/netdata-updater.sh ]; then
+		echo >&2 "Removing updater from previous location"
+		rm -f "${NETDATA_PREFIX}"/usr/libexec/netdata-updater.sh
+	fi
+
+	if [ "${INSTALLER_DIR}" ] && [ -f "${INSTALLER_DIR}/packaging/installer/netdata-updater.sh" ]; then
+		cat "${INSTALLER_DIR}/packaging/installer/netdata-updater.sh" > "${NETDATA_PREFIX}/usr/libexec/netdata/netdata-updater.sh" || exit 1
+	fi
+
+	if [ "${NETDATA_SOURCE_DIR}" ] && [ -f "${NETDATA_SOURCE_DIR}/packaging/installer/netdata-updater.sh" ]; then
```

Comment:
kickstart-static64.sh doesn't define `INSTALLER_DIR` and `NETDATA_SOURCE_DIR`. Has this been tested?

Reply:
@chris, This is the reason I changed the logic within `install_or_remove_netdata_updater` compared to what it was in `netdata-installer.sh`

Before it was
```
if [ -f "${INSTALLER_DIR}/packaging/installer/netdata-updater.sh" ]; then
```

Now it is
```
if [ "${INSTALLER_DIR}" ] && [ -f "${INSTALLER_DIR}/packaging/installer/netdata-updater.sh" ]; then
```

I have also tested this by running the installation on a clean VM

Reply:
Help me understand. Since both variables are empty, how do we end up with a file in `${NETDATA_PREFIX}/usr/libexec/netdata/netdata-updater.sh` when calling this from `kickstart-static64.sh`?

Reply:
When using kickstart.sh the netdata-updater.sh script is not installed by make install, so this function has to copy it.

But when using kickstart-static64.sh the full .tar.gz is extracted under /opt/netdata (which is NETDATA_PREFIX in this case), and the netdata-updater.sh script is already under NETDATA_PREFIX.